apps/web/lib/services/blockchain/__tests__/performance.test.ts

```
@@ -0,0 +1,342 @@
+        const user = users[i];
+        const badgeCount = Math.floor(Math.random() * maxBadgesPerUser) + 1;
+        
+        for (let j = 1; j <= badgeCount; j++) {
+          await services.badgeTracker.mint_badge(
+            user,
+            'ChapterCompletion' as BadgeType,
+            j,
+            `Badge ${j}`
+          );
+        }
+      }
```

Biome (2.1.2)

[error] Use for-of loop instead of a for loop.



(lint/style/useForOf)